js/sri-audit.js

```diff
@@ -9,23 +9,25 @@ const { v4: uuid } = require('uuid');
  * @typedef {import('sri4node')} TSri4Node
  * @typedef {import('sri4node').TSriConfig} TSriConfig
  * @typedef {import('sri4node').TPluginConfig} TPluginConfig
+ *
+ * @typedef {import('./sri-audit.d.ts').TSri4NodeAuditBroadcastPluginConfig} TSri4NodeAuditBroadcastPluginConfig
  */
 
 /**
  * 
- * @param {*} tx 
- * @param {*} pluginConfig 
- * @param {*} sriRequest 
- * @param {*} elements 
- * @param {*} component 
- * @param {*} operation 
- * @param {*} mapping 
+ * @param {any} tx databse transaction from pg-promise
+ * @param {TSri4NodeAuditBroadcastPluginConfig} pluginConfig
+ * @param {import('sri4node').TSriRequest} sriRequest
+ * @param {Array<{ permalink: string; incoming: Record<string, any>; stored: Record<string, any>;}>} elements
+ * @param {string} component
+ * @param {'CREATE' | 'READ' | 'UPDATE' | 'DELETE'} operation
```

Comment:
i don't assume READ is a valid option here?

Reply:
Fixed